instance/models/instance.py

```diff
@@ -909,6 +465,7 @@ def provision(self):
             log, exit_code = self.deploy()
             if exit_code != 0:
                 server.update_status(provisioning=True, failed=True)
+                self.provision_failed_email(self.ProvisionMessages.PROVISION_ERROR)
```

Comment:
@omarkhan Good catch - I planned to do so initially, but that completely forgot about it. To save some time, my approach is to add log as an attached file (including it in the body will further bloat the message) - do you have any objections/suggestions/comments to doing so?